src/components/CropSelector/CropTable.js

```diff
@@ -199,143 +200,141 @@ const CropTableComponent = (props) => {
-                            </Typography>
-                          </td>
-                          <td>
-                            <Typography
-                              variant="subtitle2"
-                              component="b"
-                              className="text-uppercase"
-                            >
-                              {crop.fields['Duration'].toString().toLowerCase() ===
-                              'short-lived perennial'
-                                ? 'Perennial'
-                                : crop.fields['Duration'].toString()}
-                            </Typography>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table>
-                  </TableCell>
-                  {getCardFlex(crop, index)}
-                </TableRow>
-              </Fragment>
-            );
-          else {
-            return '';
-          }
-        })
-      : null;
+                      )}
+
+                      <tr>
+                        <td>
+                          {' '}
+                          <Typography variant="subtitle2" component="b" className="">
+                            DRY MATTER:
+                          </Typography>
+                        </td>
+                        <td>
+                          <Typography variant="subtitle2" component="b">
+                            {crop.fields['Dry Matter Min (lbs/A/y)']}-
+                            {crop.fields['Dry Matter Max (lbs/A/y)']}
+                            <span className="units">lbs/A/y</span>
+                          </Typography>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <Typography variant="subtitle2" component="b" className="">
+                            DURATION:
+                          </Typography>
+                        </td>
+                        <td>
+                          <Typography variant="subtitle2" component="b" className="text-uppercase">
+                            {crop.fields['Duration'].toString().toLowerCase() ===
+                            'short-lived perennial'
+                              ? 'Perennial'
+                              : crop.fields['Duration'].toString()}
+                          </Typography>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </TableCell>
+                {getCardFlex(crop, index)}
+              </TableRow>
+            </Fragment>
+          );
+        // else {
+        //   return '';
+        // }
+      })
+    );
   };
 
   const RenderActiveInactiveCropData = () => {
     return (
-      <Fragment>
+      <>
         <CropList matchGoals={true} />
         <CropList matchGoals={false} />
-      </Fragment>
+      </>
     );
   };
 
```

Comment:
on line 341. we should be setting all state hooks at the top just under the constructor.  please review the file and try to clean this up.